juno/assetlist.json

```diff
@@ -2049,6 +2049,31 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/arena.png"
         }
       ]
+    },
+        {
+      "denom_units": [
+        {
+          "denom": "factory/juno1h6y8tkceau4d8zyv5aa0fwdj2pa2y0gz2hx0tq/uwind",
+          "exponent": 0
+        },
+        {
+          "denom": "wind",
+          "exponent": 6
+        }
+      ],
+      "address": "juno1h6y8tkceau4d8zyv5aa0fwdj2pa2y0gz2hx0tq",
+      "base": "factory/juno1h6y8tkceau4d8zyv5aa0fwdj2pa2y0gz2hx0tq/uwind",
+      "name": "Wind Token",
+      "display": "wind",
+      "symbol": "WIND",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/wind.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/wind.png"
```

Comment:
doesn't match uploaded image title (uwind vs wind). Also the image quality is really poor, not centered, and looks round but doesn't use transparency. Can we improve this please?